GetStream/stream-chat-android
# [CAS-1231] Allow passing multiple attachment's types for searching

https://stream-io.atlassian.net/browse/CAS-1231

### 🎯 Goal

Allow passing multiple attachment's types for searching

### 🛠 Implementation details

- Deprecated old methods with a single `type` parameter
- Introduced new methods with `types`

### 🧪 Testing
1. Check if you can view photos and files in the sample (ChatInfo -> Shared Photo/Files)
2. Try to search for messages wit multiple attachment `types`

### ☑️ Checklist
